### PR TITLE
Don't pre-pop make love dialog with currentUser's name

### PR DESCRIPTION
**Pull request by bschmeck**
The API doesn't let you send love to yourself

### CODE REVIEW

File: app/src/main/java/org/ometa/lovemonster/ui/activities/UserLoveActivity.java [hunk 1]
```diff
@@ -75,7 +75,12 @@ private void setupMakeLoveButton() {
         makeLoveButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                final MakeLoveDialogFragment makeLoveDialogFragment = MakeLoveDialogFragment.newInstance(user);
+                final MakeLoveDialogFragment makeLoveDialogFragment;
+                if (user.equals(currentUser)) {
+                    makeLoveDialogFragment = MakeLoveDialogFragment.newInstance(new User("foo@example.com", ""));
```
Comment (ometa) [inline app/src/main/java/org/ometa/lovemonster/ui/activities/UserLoveActivity.java:80]: Is the `foo` part just a placeholder that gets overwritten?

Comment (bschmeck) [inline app/src/main/java/org/ometa/lovemonster/ui/activities/UserLoveActivity.java:80]: An email address is required to create a User object, but the dialog only
uses the username part of the object.  So we put an empty string in the To:
field.  This is the same thing we do in LovesListActivity.

Comment (AnthonyCaliendo) [inline app/src/main/java/org/ometa/lovemonster/ui/activities/UserLoveActivity.java:80]: That constraint is probably overly restrictive, considering we only ever really need a username for stuff to work.

Probably is safe to remove the requirement on email address being there, and only require username.

Comment (bschmeck) [inline app/src/main/java/org/ometa/lovemonster/ui/activities/UserLoveActivity.java:80]: Agreed.  But, it's 11 o'clock :)

**Pull request merged**